x-pack/plugins/security_solution/public/management/components/package_action_item/package_action_formatter.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import type { DocLinks } from '@kbn/doc-links';
+
+type PackageActions = 'es_connection';
+
+export const titles = Object.freeze(
+  new Map<PackageActions, string>([
+    [
+      'es_connection',
+      i18n.translate('xpack.securitySolution.endpoint.details.packageActions.es_connection.title', {
+        defaultMessage: 'Elasticsearch Connection Failure',
```

Comment:
Sentence case for titles:
```suggestion
        defaultMessage: 'Elasticsearch connection failure',
```